<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainInterfaceClass</class>
 <widget class="QMainWindow" name="MainInterfaceClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>947</width>
    <height>777</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Контакты Отдела Закупок</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="searchGBox">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Поиск</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>12</number>
       </property>
       <item>
        <widget class="QLineEdit" name="searchLineEdit">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="position_rBtn">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Позиция</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="supplier_rBtn">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Поставщик</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="suppliersTable">
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::SelectedClicked</set>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::NoDragDrop</enum>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="verticalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="horizontalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>_id</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Поставщик</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Статус</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>12</number>
      </property>
      <item>
       <widget class="QTableWidget" name="positionsTable">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="verticalScrollMode">
         <enum>QAbstractItemView::ScrollPerPixel</enum>
        </property>
        <property name="horizontalScrollMode">
         <enum>QAbstractItemView::ScrollPerPixel</enum>
        </property>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>_id</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Позиция</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTableWidget" name="contactsTable">
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="textElideMode">
           <enum>Qt::ElideMiddle</enum>
          </property>
          <property name="verticalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="horizontalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <column>
           <property name="text">
            <string>_id</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Имя</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Телефон</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>E-mail</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="remarkGBox">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>120</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Примечание</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QTextEdit" name="remarkTextEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>80</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="close_btn">
        <property name="text">
         <string>Закрыть</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>947</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="Options_menu">
    <property name="title">
     <string>Настройки</string>
    </property>
    <widget class="QMenu" name="showActiveSupp_action">
     <property name="title">
      <string>Сортировка</string>
     </property>
     <addaction name="isShowing_action"/>
    </widget>
    <widget class="QMenu" name="database_menu">
     <property name="title">
      <string>База данных</string>
     </property>
     <addaction name="setPathToDB_action"/>
     <addaction name="showPathToDB_action"/>
    </widget>
    <addaction name="database_menu"/>
    <addaction name="showActiveSupp_action"/>
   </widget>
   <addaction name="Options_menu"/>
  </widget>
  <action name="isShowing_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Показывать неактивных поставщиков</string>
   </property>
  </action>
  <action name="setPathToDB_action">
   <property name="text">
    <string>Указать базу данных</string>
   </property>
  </action>
  <action name="showPathToDB_action">
   <property name="text">
    <string>Текущий путь к базе данных</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MainInterface.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>suppliersTable</sender>
   <signal>itemPressed(QTableWidgetItem*)</signal>
   <receiver>MainInterfaceClass</receiver>
   <slot>selectSupplierSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>152</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>86</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>position_rBtn</sender>
   <signal>released()</signal>
   <receiver>MainInterfaceClass</receiver>
   <slot>searchSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>331</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>433</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>supplier_rBtn</sender>
   <signal>released()</signal>
   <receiver>MainInterfaceClass</receiver>
   <slot>searchSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>421</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>616</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>searchLineEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>MainInterfaceClass</receiver>
   <slot>searchSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>24</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>close_btn</sender>
   <signal>clicked()</signal>
   <receiver>MainInterfaceClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>453</x>
     <y>733</y>
    </hint>
    <hint type="destinationlabel">
     <x>722</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>testSlot()</slot>
  <slot>selectSupplierSlot()</slot>
  <slot>changeSearchesObjectSlot()</slot>
  <slot>searchSlot()</slot>
 </slots>
</ui>
